core/CHANGELOG.md

```diff
@@ -5,12 +5,14 @@
 - Drop `Unpin` requirement from `SubstreamBox`. See [PR 2762] and [PR 2776].
 - Drop `Sync` requirement on `StreamMuxer` for constructing `StreamMuxerBox`. See [PR 2775].
 - Use `Pin<&mut Self>` as the receiver type for all `StreamMuxer` poll functions. See [PR 2765].
+- Generalise `StreamMuxer::poll_address_change` to `StreamMuxer::poll_event`. See [PR XXXX].
```

Comment:
This is a weird changelog entry because we haven't even released the top one yet but we are re-introducing a function here that was removed in a PR that is contained in the same release. How would you like to handle this @mxinden?

Reply:
I would suggest we update the entry introduced in https://github.com/libp2p/rust-libp2p/pull/2724/.